src/utils/flagCodes.ts

```diff
@@ -7,8 +7,20 @@ export interface Flag<T extends string> {
 }
 
 // OUR specific flags
+const FIXED_SENTINEL = 'dec0de'
+
+const APP_ID = Number(process.env.APP_ID || CONFIG.transactions.appId)
+if (!Number.isInteger(APP_ID) || APP_ID < 1 || APP_ID > 99) {
+  throw new Error(`APP_ID isn't valid. Expected a positive integer <= 99, got ${APP_ID}`)
```

Comment:
```suggestion
  throw new Error(`APP_ID isn't valid. Expected a positive integer >=1 and <= 99, got ${APP_ID}`)
```

Reply:
This is changing anyway with @anxolin's PR #1267